### ISSUE TITLE
Sample header line (starting with "#CHROM") does not start with required prefix #CHROM

### ISSUE THREAD

**Issue by InduKhatri** [griffithlab/pVACtools#833]
* pvactools version: 
* Python version:
* Operating System: 

Docker Container: latest: 3.0.4

**Describe the bug**
The VCF file is not read correctly. Although the file has #CHROM header the tool cant read it.

**To Reproduce**
```
pvacseq run -e1 8,9,10 -e2 15 /pvactools_input_data/input.vcf H_NJ-HCC1395-HCC1395 HLA-A*02:01,HLA-B*35:01,DRB1*11:01 NetMHC /pvactools_output_data/
```

**Log Output**
```
With the example INPUT file provided by pVACSeq, I get the following error
Executing MHC Class I predictions
Converting .vcf to TSV
TSV file already exists. Skipping.
Converting VCF to TSV
Traceback (most recent call last):
  File "/usr/local/bin/pvacseq", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.7/site-packages/pvactools/tools/pvacseq/main.py", line 116, in main
    args[0].func.main(args[1])
  File "/usr/local/lib/python3.7/site-packages/pvactools/tools/pvacseq/run.py", line 131, in main
    pipeline.execute()
  File "/usr/local/lib/python3.7/site-packages/pvactools/lib/pipeline.py", line 464, in execute
    self.generate_combined_fasta(self.fasta_file_path())
  File "/usr/local/lib/python3.7/site-packages/pvactools/lib/pipeline.py", line 179, in generate_combined_fasta
    generate_combined_fasta.main(params)
  File "/usr/local/lib/python3.7/site-packages/pvactools/tools/pvacseq/generate_protein_fasta.py", line 161, in main
    proximal_variants_tsv = convert_vcf(args.input_vcf, temp_dir, args.sample_name, args.phased_proximal_variants_vcf, args.flanking_sequence_length, args.pass_only)
  File "/usr/local/lib/python3.7/site-packages/pvactools/tools/pvacseq/generate_protein_fasta.py", line 89, in convert_vcf
    converter = VcfConverter(**convert_params)
  File "/usr/local/lib/python3.7/site-packages/pvactools/lib/input_file_converter.py", line 79, in __init__
    self.vcf_reader = vcfpy.Reader.from_path(self.input_file)
  File "/usr/local/lib/python3.7/site-packages/vcfpy/reader.py", line 99, in from_path
    parsed_samples=parsed_samples,
  File "/usr/local/lib/python3.7/site-packages/vcfpy/reader.py", line 65, in from_stream
    parsed_samples=parsed_samples,
  File "/usr/local/lib/python3.7/site-packages/vcfpy/reader.py", line 121, in __init__
    self.header = self.parser.parse_header(parsed_samples)
  File "/usr/local/lib/python3.7/site-packages/vcfpy/parser.py", line 733, in parse_header
    self.samples = self._handle_sample_line(parsed_samples)
  File "/usr/local/lib/python3.7/site-packages/vcfpy/parser.py", line 769, in _handle_sample_line
    self._check_samples_line(arr)
  File "/usr/local/lib/python3.7/site-packages/vcfpy/parser.py", line 784, in _check_samples_line
    "start with required prefix {}".format("\t".join(REQUIRE_SAMPLE_HEADER))
vcfpy.exceptions.IncorrectVCFFormat: Sample header line (starting with "#CHROM") does not start with required prefix #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT
```

**Comment by susannasiebert** [comment]
Hi @InduKhatri,

Unfortunately, I'm unable to reproduce this error. There is no official 3.0.4 release so I tried with the griffithlab/pvactools:3.0.3-xs docker image, which works fine. Can you please list the full docker image tag you are using as well as how you are retrieving the sample data?

Here is what I did:
```
$ docker run -it griffithlab/pvactools:3.0.3-xs

root@83fef75a69af:/# pvacseq download_example_data .

root@83fef75a69af:/# pvacseq run -e1 8,9,10 -e2 15 pvacseq_example_data/input.vcf H_NJ-HCC1395-HCC1395 HLA-A*02:01,HLA-B*35:01,DRB1*11:01 NetMHC /pvactools_output_data/
```

**Comment by InduKhatri** [comment]
Hi Susanna,

Thank you for your quick response.
The VCFParser was throwing this error also with my own files. I readjusted the header and removed some lines for the header and it worked fine.

Thank you again.

**Comment by susannasiebert** [comment]
Glad to hear you were able to resolve this issue. Could you please provide more details how you adjusted the header? This would help other users who might run into this error in the future.

**Comment by ZoeChao2001** [comment]
Hi InduKhatri，
I have also encountered this issue. Could you please provide more details how you adjusted the header? I‘ll really appreciate it if you could response me.

**Comment by susannasiebert** [comment]
Hi @ZoeChao2001, if you can post your VCF header line here, I can also investigate and hopefully give you some pointers.

**Comment by InduKhatri** [comment]
Hi,
Sorry that I could not reply earlier. I searched for the code I used to clean the VCF file but unfortunately could not find it.

My VCF file had a big header with sample information along with an additional header from the VEP plugins I used. pVACtools had an issue with such extensive and most likely also redundant information in the header in my case, therefore, I removed everything from the header except the VCF version (first line) and #CHROM line then used pVACtools. 

In case you used VEP plugins, it would be nice to keep the header information for the plugin as well.

Overall, I kept the explanation regarding the columns and their content, but not the sample information in the header.

I hope it solves the issue for you. Good luck!
